dbt/README.md

```diff
@@ -222,11 +199,8 @@ You should almost never have to manually build tables and views in our
 production environment, since this repository is configured to automatically
 deploy production models using GitHub Actions for continuous integration.
 However, in the rare case that you need to manually build models in production,
-use the `--target` option:
-
-```
-dbt build --target prod --resource-types model seed
-```
```

Comment:
Folks should be using the GitHub workflow to rebuild prod models, not running `dbt build --target prod` in the command line like a chaos monkey.